src/pyscaffold/templates/github_ci_workflow.template

```diff
@@ -0,0 +1,114 @@
+# GitHub Actions configuration **EXAMPLE**,
+# MODIFY IT ACCORDING TO YOUR NEEDS!
+# Reference: https://docs.github.com/en/actions
+
+name: tests
+
+on:
+  push:
+    # Avoid using all the resources/limits available by checking only
+    # relevant branches and tags. Other branches can be checked via PRs.
+    branches: [main]
+    tags: ['v*']  # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:  # Run in every PR
+  workflow_dispatch:  # Allow manually triggering the workflow
+  schedule:
+    # Run roughly every 15 days at 00:00 UTC
+    # (useful to check if updates on dependencies break the package)
+    - cron: '0 0 1,16 * *'
+
+concurrency:
+  group: >-
+    ${{ github.workflow }}-${{ github.ref_type }}-
+    ${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      wheel-distribution: ${{ steps.wheel-distribution.outputs.path }}
+    steps:
+      - uses: actions/checkout@v2
+        with: {fetch-depth: 0}  # deep clone for setuptools-scm
+      - uses: actions/setup-python@v2
+        with: {python-version: "3.10"}
+      - name: Lint code
+        run: pipx run tox -e lint
```

Comment:
Here we can just `pipx run pre-commit --all-files` instead (considering that the `testenv:lint` environment is not available).

Reply:
Ok, then it's also clearer what happens and one doesn't need to look up what `lint` is mapped to. On the other hand, I got used to calling `tox -e build`, and using `tox` for all kinds of runner tasks is also fine. So both is fine for me. 
Why is the `testenv:lint` not available?